README.md

```diff
@@ -44,7 +44,271 @@ To build, ensure docker is installed, and run:
 
 ### Deploy a GD2 gluster cluster
 
-### Deploy CSI driver
+### 1. RW0 Volume Claim (Gluster Virtual Block CSI driver)
+
+```
+[root@localhost]#kubectl create -f csi-deployment.yaml
+service/csi-attacher-glustervirtblockplugin created
+statefulset.apps/csi-attacher-glustervirtblockplugin created
+daemonset.apps/csi-nodeplugin-glustervirtblockplugin created
+service/csi-provisioner-glustervirtblockplugin created
+statefulset.apps/csi-provisioner-glustervirtblockplugin created
+serviceaccount/glustervirtblock-csi created
+clusterrole.rbac.authorization.k8s.io/glustervirtblock-csi created
+clusterrolebinding.rbac.authorization.k8s.io/glustervirtblock-csi-role created
+```
+
+>NOTE: You can skip seperate installation of kubernetes cluster, GD2 Cluster
+and CSI deployment if you directly use [GCS](https://github.com/gluster/gcs)
+installation method, it should bring your deployment in one shot. Refer
+[GCS deployment guide](https://github.com/gluster/gcs/blob/master/deploy/README.md)
+for more details.
+
+### Create a gluster virtual block storage class
+
+```
+[root@localhost ~]$ cat storage-class.yaml
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: glustervirtblock-csi
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: org.gluster.glustervirtblock
+
+[root@localhost]# kubectl create -f storage-class.yaml
+storageclass.storage.k8s.io/glustervirtblock-csi created
+```
+
+Verify gluster virtual block storage class
+
+```
+[root@localhost]# kubectl get storageclass
+NAME                             PROVISIONER                    AGE
+glusterfs-csi (default)          org.gluster.glusterfs          28h
+glustervirtblock-csi (default)   org.gluster.glustervirtblock   6s
+local-storage                    kubernetes.io/no-provisioner   29h
+```
+
+Unset glusterfs-csi as default CSI driver if it's default
```

Comment:
ah, this is not required please make gluster-block storage class as non default one

Reply:
Done